### ISSUE TITLE
neres login fails with `ImportError: No module named yamlordereddictloader`

### ISSUE THREAD

**Issue by bookshelfdave** [glogiotatidis/neres#4]
* neres version: 0.3.0
* Python version: 2.7.13
* Operating System: MacOS Sierra 10.12.5

### Description

Missing dep `yamlordereddictloader` causes `neres login` to fail. 

`pip install yamlordereddictloader` resolved the issue for me.

### What I Did

```
$ neres login
Traceback (most recent call last):
  File "/usr/local/bin/neres", line 7, in <module>
    from neres.cli import cli
  File "/usr/local/lib/python2.7/site-packages/neres/cli.py", line 13, in <module>
    import yamlordereddictloader
ImportError: No module named yamlordereddictloader
10:59:59

$ pip install yamlordereddictloader
Collecting yamlordereddictloader
  Downloading yamlordereddictloader-0.2.2.tar.gz
Requirement already satisfied: pyyaml in ./Library/Python/2.7/lib/python/site-packages (from yamlordereddictloader)
Building wheels for collected packages: yamlordereddictloader
  Running setup.py bdist_wheel for yamlordereddictloader ... done
  Stored in directory: /Users/dparfitt/Library/Caches/pip/wheels/cf/7c/9d/1c727b69488b8639e7408f4f8f947ffce71cf824883daf9e04
Successfully built yamlordereddictloader
Installing collected packages: yamlordereddictloader
Successfully installed yamlordereddictloader-0.2.2

$ neres login
Email: ^CAborted!
```

**Comment by glogiotatidis** [comment]
Thanks for reporting @metadave. This should be fixed in 1149cc71d252f55ecee6f3743c7266eacf2fa816

**Comment by glogiotatidis** [comment]
Thanks for reporting @metadave. This should be fixed in 1149cc7

**Comment by glogiotatidis** [comment]
Released `v0.3.1` with the fix, `pip install -U neres` to get it
